doc/General/Callback-Stats-and-Privacy.md

```diff
@@ -11,12 +11,12 @@ Now onto the bit you're interested in, what is submitted and what we do with tha
 
 #### What is submitted ####
 - All data is anonymous.
-- Generic statistics are taken from the database, these include things like device total, type and os, port types, speeds and total, total bgp peers. Take a look at the code for full details.
```

Comment:
Oops lost something here.  This meant port types, port speeds, and port counts **AND** BGP peer count.  Now it doesn't mention port count.

Should replace total with count in my opinion.

Reply:
@murrant Ah I see. I changed it again, would be glad if you could take a look at it (also I hope I didn't tag you too many times, the comment on the other request was a mistake...)